README.md

```diff
@@ -126,9 +138,6 @@ Documentation on how to build custom transformers to work with these commands ca
 - `make test` will run the unit tests and skip the integration tests
 - `make integrationtest` will run just the integration tests
 
-## API
-[Postgraphile](https://www.graphile.org/postgraphile/) is used to expose GraphQL endpoints for our database schemas, this is described in detail [here](../staging/postgraphile/README.md).
-
 
 ## Contributing
 Contributions are welcome! For more on this, please see [here](../staging/documentation/contributing.md).
```

Comment:
I would love to include a Code of Conduct in here, for when we receive community contributions. The [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) is one that comes to mind, but I'm certainly happy to do some more research/work on finding one that fits Vulcanize.

Reply:
8dc9ddf65e4004bcb430892594f4cc999a0e6393